app/models/keyword.rb

```diff
@@ -11,4 +11,10 @@ class Keyword < ApplicationRecord
             presence: true,
             numericality: { greater_than_or_equal_to: 0 },
             if: :succeeded?
+
+  after_create_commit :enqueue_scrape_job
```

Comment:
did you check this on your local, technically this branch need to be checkout from the `https://github.com/Goose97/google-scraper-ruby/pull/31` so you could test the whole flow there..

Reply:
yeah, I tested against #31 in my local setup. This PR depends on #31, hence the draft status. Once #31 is merged, I will have the whole flow tested again